src/common/styles/cd-filler-loading.less

```diff
@@ -8,16 +8,21 @@
     position: absolute;
     height: 100%;
     width: 100%;
-    animation: fillerLoading .8s linear infinite;
-    background: linear-gradient(to right, transparent 42%, fade(@cd-white, 50%) 50%, transparent 58%) 50% 50%;
+    animation: fillerLoading .8s ease-in infinite;
+    background: linear-gradient(to right, transparent calc(~"50% - 30px"), fade(@cd-white, 50%) 50%, transparent calc(~"50% + 30px")) 50% 50%;
+  }
+  &--grey-bg {
```

Comment:
Couldn't have it be possible to have a function that takes a color ?
I guess we don't have many possible backgrounds, so that may be overkill ?

Reply:
There's also the question about how it would look in different colours. The grey bg used in the sidebar just happens to work. Different shades might not.